workflows_api/runtime/src/schemas.py

```diff
@@ -280,3 +288,6 @@ def only_one_discover_item(cls, discovery_items):
                 "Zarr dataset should include zarr_store in its discovery item"
             )
         return discovery_items
+
+    class Config:
+        extra = Extra.allow
```

Comment:
does this not get inherited from `Dataset` ?

Reply:
makes sense. removing it.

Reply:
makes sense. Removing it!